pkg/microservice/aslan/core/common/service/config_payload.go

```diff
@@ -66,7 +66,8 @@ func GetConfigPayload(codeHostID int) *models.ConfigPayload {
 		JenkinsBuildConfig: models.JenkinsBuildConfig{
 			JenkinsBuildImage: config.JenkinsImage(),
 		},
-		AesKey: crypto.GetAesKey(),
+		AesKey:           crypto.GetAesKey(),
+		BuildConcurrency: 5,
```

Comment:
Use `concurrencySettings.BuildConcurrency`?

Reply:
This is the default value, in case something bad happens in the logic below